tsdb/index/tsi1/tsi1.go

```diff
@@ -802,6 +802,15 @@ func copyBytes(b []byte) []byte {
 	return buf
 }
 
+// copyBytes returns a copy of a slice of byte slices.
+func copyByteSlices(a [][]byte) [][]byte {
```

Comment:
I don't have any qualms about this being here, but it could also live happily alongside `Union(a, b [][]byte) [][]byte` and `Intersect(a, b [][]byte) [][]byte` in the `bytesutil` package. Maybe as `Clone(a [][]byte) [][]byte`?

Reply:
Good point. I moved these up to `bytesutil` in d17d0f1.